app/controllers/conference_controller.rb

```diff
@@ -5,7 +5,7 @@ class ConferenceController < ApplicationController
   load_resource :program, through: :conference, singleton: true, except: :index
 
   def index
-    @current = Conference.where('end_date >= ?', Date.current).order('start_date ASC')
+    @current = Conference.where('end_date >= ?', Date.current).reorder('start_date ASC')
```

Comment:
Yes I think so too. Isn't that what you are trying to do? Now it's `ASC` and the wrong order right? :-)